medmodels/medrecord/querying.pyi

```diff
@@ -175,19 +179,62 @@ class SingleValueOperand:
     ) -> None: ...
     def clone(self) -> SingleValueOperand: ...
 
+class AttributesTreeOperand:
+    def max(self) -> MultipleAttributesOperand: ...
+    def min(self) -> MultipleAttributesOperand: ...
+    def count(self) -> MultipleAttributesOperand: ...
+    def sum(self) -> MultipleAttributesOperand: ...
+    def first(self) -> MultipleAttributesOperand: ...
+    def last(self) -> MultipleAttributesOperand: ...
+    def is_string(self) -> None: ...
+    def is_int(self) -> None: ...
+    def is_max(self) -> None: ...
+    def is_min(self) -> None: ...
+    def greater_than(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def greater_than_or_equal_to(
+        self, value: SingleAttributeComparisonOperand
+    ) -> None: ...
+    def less_than(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def less_than_or_equal_to(
+        self, value: SingleAttributeComparisonOperand
+    ) -> None: ...
+    def equal_to(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def not_equal_to(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def is_in(self, values: MultipleAttributesComparisonOperand) -> None: ...
+    def is_not_in(self, values: MultipleAttributesComparisonOperand) -> None: ...
+    def starts_with(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def ends_with(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def contains(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def add(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def subtract(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def multiply(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def modulo(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def power(self, value: SingleAttributeComparisonOperand) -> None: ...
+    def absolute(self) -> None: ...
```

Comment:
no sqrt, round, ceil or floor?

Reply:
disregard. it should also not have a `round`, `ceil` and `floor`. A `MedRecordAttribute` is either a `string` or a `int`. On strings these operations would not do anything and on an `int` as well since an integer is always rounded